Adds workaround for building against stable VS.

zooba opened the pull request:
This will keep us building with the current version of VS while we figure out how to deal with the brokenness in the next update.

We'll just have to ignore the failed tests for now until we can get all the build machines upgraded properly (which they probably won't do for us, so we're back to maintaining our own build infrastructure...)

zooba commented:
I've already updated the build definitions to use a working build pool, so feel free to merge this whenever and we'll get tests going again.

huguesv commented:
What do you mean by working build pool?  Is it machines that we manage ourselves that are currently on 15.2 and that we'll update to 15.3 shortly?

zooba commented:
No, we switched to the one that is managed by the build team a while back, but they won't update to 15.3 until after it releases. I upgraded our own one and switched back to that briefly, but it won't build for a variety of other reasons and I don't want to spend the time debugging that when we could (and should) just be getting the editor team to fix their API breakage.
